cmd/lotus-miner/init_service.go

```diff
@@ -105,7 +105,7 @@ var serviceCmd = &cli.Command{
 			if es.Contains(MarketsService) {
 				log.Info("Configuring miner actor")
 
-				if err := configureStorageMiner(ctx, api, maddr, peerid, big.Zero()); err != nil {
+				if err := configureStorageMiner(ctx, api, maddr, peerid, big.Zero(), cctx.Uint64("confidence")); err != nil {
```

Comment:
We need to add the flag to `serviceCmd` (and also to `restoreCmd`)

Reply:
sercviceCmd and restoreCmd is subcommand of init.  specific this flag in parent context also work
```bash
./lotus-miner  init --confidence 10 restore
```